WifiART/src/userinterface/ArtMvcView.java

```diff
@@ -30,13 +38,45 @@
     private static final String LCR_TAB_NAME = "Loc. Civic";
     private static final String MAP_TAB_NAME = "Map Image";
 
+    //Generate Tab labels
+    private final JLabel lciSubelementsLabel = new JLabel(" LCI Subelements Included:");
+    private final JCheckBox lciCheckbox = new JCheckBox("Location Configuration Information (LCI)");
+    private final JCheckBox zCheckbox = new JCheckBox("Z (floor number, height above floor)");
+    private final JCheckBox usageCheckbox = new JCheckBox("Usage Rule / Policy");
+    private final JCheckBox bssidCheckbox = new JCheckBox("BSSID List (co-located BSS's in AP)");
+    private final JLabel lcrSubelementsLabel = new JLabel(" LCR Subelements Included:                           ");
+    private final JCheckBox lcrCheckbox = new JCheckBox("Location Civic (Address)");
+    private final JCheckBox mapCheckbox = new JCheckBox("MapImage");
+    private final JLabel inputFileNameLabel = new JLabel(" Output File: ");
+    private final JTextField inputFileNameField = new JTextField();
+    private final JLabel inputDirLabel = new JLabel(" Output Dir: ");
+    private final JTextField inputDirField = new JTextField();
+    private final JLabel outputFileNameLabel = new JLabel(" Output File: ");
+    private final JTextField outputFileNameField = new JTextField();
+    private final JLabel outputDirLabel = new JLabel(" Output Dir: ");
+    private final JTextField outputDirField = new JTextField();
+    private final JLabel bufferTextLabel = new JLabel("Hex Output LCI/LCR Buffer: ");
+    private final JTextArea bufferTextArea = new JTextArea();
+
+    //LCI Tab Labels
+
+    //Z Tab Labels
+
+    //Usage Tab Labels
+
+    //BSSID Tab Labels
+
+    //LCR Tab Labels
+
+    //Map Image Tab Labels
+
 
     /** Constructor.
      *
      * @param state the system state
      * */
     public ArtMvcView(ArtSystemState state) {
-        /* Top level JFRAME */
+        // Top level JFRAME
         this.setTitle("Tabbed Pane Example");
```

Comment:
Shouldn't this be the name of the top level window that a use will see?

Reply:
The top level JFrame is the ArtMvcView, as this extends JFrame. The comment is there to clarify that the following lines are adjusting the JFrame properties for the top level window. Alternatively, the ArtMvcView could have a JFrame as a property instead of being a JFrame. However, the MVC template did it the way I have implemented, which I believe is a popular way of doing it.

Reply:
If you're referring to the line "this.setTitle(...", then I agree that the placeholder should be replaced, and that will be included in the revision.

Reply:
Yes - the title needs to be updated.